docs/workspaces.md

```diff
@@ -577,13 +580,6 @@ only available to Nodes within *one* Availability Zone. There is usually an opti
 but they have trade-offs, e.g. you need to pay for multiple volumes since they are replicated and your volume may have 
 substantially higher latency.
 
-When using a workspace backed by a `PersistentVolumeClaim` (typically only available within a Data Center) and the `TaskRun`
```

Comment:
Why remove this, as opposed to updating to mention the new coscheduling options?

Reply:
I rephrased this in the new section below

Reply:
Maybe link to that new section here?

Reply:
I just move the content to a new section, so I don't think a new section is needed